recipes/refl-cpp/all/conanfile.py

```diff
@@ -11,6 +11,11 @@ class ReflCppConan(ConanFile):
     description = "A modern compile-time reflection library for C++ with support for overloads, templates, attributes and proxies "
     topics = ("header", "header-only", "reflection", "modern", "metaprogramming")
     no_copy_source = True
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
Seems to be the only one used

```suggestion
    settings =  "compiler"
```